sr_multi_moveit/sr_multi_moveit_config/README.md

```diff
@@ -0,0 +1,7 @@
+# sr_multi_moveit_config
+
+Configuration of multiple-robots running as a combined robot.
+
+## Multi-Robot Move Groups and States
+
+At run time, [generate_robot_srdf.py](scripts/generate_robot_srdf.py) parses individual robot (UR arms, Shadow Hands) U/SRDFs and creates move groups that include links from multiple robot definitions. It also loads and parses a configuration file ([multi_robot_move_group_states.yaml](config/multi_robot_move_group_states.yaml) by default) that defines named joint states for these combined robot move groups. These combined move group states can inherit joint angles from states defined in individual robot U/SRDFs.
```

Comment:
The only thing i'd like to see added is what is the output of `generate_robot_srdf.py` after parsing all this information.
Is it a new robot_semantic_description loaded to param server?

Reply:
Added a conclusion in https://github.com/shadow-robot/sr_interface/pull/768/commits/b59c57698fc0c4ea7c7c6dbfaff00c4e3d4fa287.

Reply:
Whoops fixed in https://github.com/shadow-robot/sr_interface/pull/768/commits/87510eed6e1d93adfd7947b648956e6c00d8db70.